### ISSUE TITLE
2 unit tests do not pass when running all tests

### ISSUE THREAD

**Issue by dvdtsr** [didomi/gtm-template#16]
Each unit test is passed when run individually, but when all tests are run 2 of them permanently fail:
`It correctly sets consent when values are set`
and
`It correctly sets consent when just one of the default states is set (storage)`

It seems that this is not due to the tests themselves (which are correct), there is probably something in the GTM test environment that misdefines the variables when triggering several successive tests.

![unit-tests](https://user-images.githubusercontent.com/13331165/142647515-857d3551-c800-43e5-96b4-7d89ed2b1cee.png)

**Comment by jawadst** [comment]
@dvdtsr I think that this is a known problem that @EdgardoRodriguezSolano is looking into

**Comment by EdgardoRodriguezSolano** [comment]
@jawadst Correct. The last thing we know about this is that the issue might be that the DOM used for testing persists across tests. I think @dvdtsr opened the issue to give visibility of this known issue to the repo users that might encounter it.

**Comment by EdgardoRodriguezSolano** [comment]
Closing as this is no longer the case after https://github.com/didomi/gtm-template/commit/100c1708c4a43f56697f22aa1fe409d909d70df2
